app/api/search.v2/searchResponse.ts

```diff
@@ -0,0 +1,41 @@
+import { EntitySchema } from 'shared/types/entityType';
+import { SearchQuery } from 'shared/types/SearchQueryType';
+
+interface ElasticSearchResults {
+  _index: string;
+  _type: string;
+  _id: string;
+  _source: EntitySchema;
+  // eslint-disable-next-line camelcase
+  inner_hits?: { fullText: { hits: { hits: [{ highlight: {} }] } } };
+}
```

Comment:
@mfacar there is already an interface defined for elastic results, can you extend that one instead of defining a new one here ? https://github.com/huridocs/uwazi/blob/db8fa58a005d771170ee9cc06f1e02a2a34e1d87/app/api/search/elasticTypes.ts#L18

Reply:
Yes, thanks, it's done